_data/other.yml

```diff
@@ -525,8 +525,12 @@ websites:
     - name: WageWorks
       url: https://www.wageworks.com
       img: wageworks.png
-      tfa: No
-      twitter: WageWorksCares
+      tfa: Yes
+      sms: Yes
+      email: Yes
+      doc: https://www.wageworks.com/media/567495/terms-of-use.pdf
```

Comment:
I don't think that document fits as a documentation. It just mentions that 2FA may be offered and what kinds of 2FA they support.
For the documentations tag we want something that describes for the user how they can enable 2FA.

Reply:
Good point. Documentation link removed.